SmartDeviceLink/private/SDLTextAndGraphicManager.m

```diff
@@ -182,7 +178,10 @@ - (void)sdl_updateAndCancelPreviousOperations:(BOOL)supersedePreviousOperations
         } else if (error != nil) {
             // Invalidate data that's different from our current screen data if a Show or SetDisplayLayout fails. This will prevent subsequent `Show`s from failing if the request failed due to the developer setting invalid data or subsequent `SetDisplayLayout`s from failing if the template is not supported on the module. 
             [strongSelf sdl_resetFieldsToCurrentScreenData];
-            [strongSelf sdl_updatePendingOperationsWithNewScreenData:strongSelf.currentScreenData];
+            SDLTextAndGraphicState *errorState = [error.userInfo objectForKey:SDLTextAndGraphicFailedScreenStateErrorKey];
+            if (errorState) {
```

Comment:
```suggestion
            if (errorState != nil) {
```

What you had is fine, but the style of the project is to do an explicit check in this case.